scene/gui/label.cpp

```diff
@@ -597,6 +625,15 @@ int Label::get_total_character_count() const {
 	return xl_text.length();
 }
 
+void Label::set_header_mode(HeaderMode p_mode) {
+	header_mode = p_mode;
+	dirty = true;
+	update();
+}
+Label::HeaderMode Label::get_header_mode() const {
+	return header_mode;
+}
+
```

Comment:
There should be an empty line between these methods.

Reply:
Thanks! I wish we could automate this..